Signal/src/ViewControllers/DebugUI/DebugUISessionState.m

```diff
@@ -81,56 +77,6 @@ - (nullable OWSTableSection *)sectionForThread:(nullable TSThread *)threadParame
                    ]];
 }
 
-+ (void)presentVerificationStatePickerForContactThread:(TSContactThread *)contactThread
-{
-    DDLogError(@"%@ Choosing verification state.", self.logTag);
-
-    NSString *title = [NSString stringWithFormat:@"Choose verification state for %@", contactThread.name];
-    UIAlertController *alertController =
-        [UIAlertController alertControllerWithTitle:title message:nil preferredStyle:UIAlertControllerStyleActionSheet];
-
-    NSString *recipientId = [contactThread contactIdentifier];
-    OWSIdentityManager *identityManger = [OWSIdentityManager sharedManager];
-
-    [alertController addAction:[UIAlertAction actionWithTitle:@"Default"
-                                                        style:UIAlertActionStyleDefault
-                                                      handler:^(UIAlertAction *_Nonnull action) {
-                                                          NSData *identityKey =
-                                                              [identityManger identityKeyForRecipientId:recipientId];
-                                                          [[OWSIdentityManager sharedManager]
-                                                               setVerificationState:OWSVerificationStateDefault
-                                                                        identityKey:identityKey
-                                                                        recipientId:recipientId
-                                                              isUserInitiatedChange:NO];
-                                                      }]];
-    [alertController addAction:[UIAlertAction actionWithTitle:@"Verified"
-                                                        style:UIAlertActionStyleDefault
-                                                      handler:^(UIAlertAction *_Nonnull action) {
-                                                          NSData *identityKey =
-                                                              [identityManger identityKeyForRecipientId:recipientId];
-                                                          [[OWSIdentityManager sharedManager]
-                                                               setVerificationState:OWSVerificationStateVerified
-                                                                        identityKey:identityKey
-                                                                        recipientId:recipientId
-                                                              isUserInitiatedChange:NO];
-                                                      }]];
-    [alertController addAction:[UIAlertAction actionWithTitle:@"No Longer Verified"
```

Comment:
This skips some important machinery - if you want someone to be marked as no-longer-verified, first verify them in the fingerprint controller, then use the debug UI to toggle a SN change.